[2022-03-17 03:18:49,217] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-03-02T06:00:00+00:00 [queued]>
[2022-03-17 03:18:49,260] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-03-02T06:00:00+00:00 [queued]>
[2022-03-17 03:18:49,262] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:18:49,263] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:18:49,265] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:18:49,296] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-03-02T06:00:00+00:00
[2022-03-17 03:18:49,301] {standard_task_runner.py:52} INFO - Started process 737 to run task
[2022-03-17 03:18:49,322] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-03-02T06:00:00+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmp_8pau4cl', '--error-file', '/tmp/tmpo7j8krym']
[2022-03-17 03:18:49,330] {standard_task_runner.py:77} INFO - Job 8: Subtask ingest
[2022-03-17 03:18:49,453] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-03-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:18:49,577] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T06:00:00+00:00
[2022-03-17 03:18:49,580] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:18:49,584] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:18:49,603] {bash.py:169} INFO - Output:
[2022-03-17 03:18:49,686] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:18:49,688] {bash.py:173} INFO - dags
[2022-03-17 03:18:49,691] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:18:49,693] {bash.py:173} INFO - logs
[2022-03-17 03:18:49,695] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:18:49,696] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:18:49,698] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:18:49,699] {bash.py:173} INFO - plugins
[2022-03-17 03:18:49,701] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:18:49,702] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:18:49,703] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:18:49,760] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190302T060000, start_date=20220317T031849, end_date=20220317T031849
[2022-03-17 03:18:49,835] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:18:49,884] {local_task_job.py:146} INFO - Task exited with return code 0
